### DISCUSSION TITLE
Disabling transition for web only ?

### DISCUSSION THREAD

**Discussion by fvisticot** [csells/go_router#203] [Q&A]
Is it possible to disable transition for web only ?

**Comment by csells** [comment]
You bet. You have complete control over how transitions work in go_router as per: https://gorouter.dev/custom-transitions

In this case `kIsWeb` is your friend.

**Reply by fvisticot** [reply]
I read this documentation but I was wondering if a global go_router option was available for customizing transition for all routes of the router (only for web)
It will allow me to make the configuration with one line instead of configuring each route of the router.

**Comment by csells** [comment]
not yet but I could imagine such a thing...

**Reply by fvisticot** [reply]
Tx for the quick reply and I vote for this new feature :) ... if it makes sense !

**Comment by csells** [comment]
https://github.com/csells/go_router/issues/204
